my-vue3-app/src/services/api.ts

```diff
@@ -1,18 +1,29 @@
 // should be imported from API or somewhere else
+
 export interface Report {
   id: string;
   title: string;
   nick: string;
   gpgSignature: string;
 }
+
+function extractNick(filters: object) {
+  return Object.values(filters);
+  // return nick
+}
+
 export default {
-  async fetchReports({ page, pageSize }: {page: number; pageSize: number}): Promise<{
+  async fetchReports(
+    { page, pageSize, filters }:
+    {page: number; pageSize: number; filters: object},
```

Comment:
`filters: object` is bad practice. just make it obvious and explicit. `filters: { ... }`

Reply:
in the future we will change it to worse but still we will have operators and there will be some structure not `object`